youtube_dl/extractor/expressen.py

```diff
@@ -28,12 +28,38 @@ class ExpressenIE(InfoExtractor):
     }, {
         'url': 'https://www.expressen.se/tv/kultur/kulturdebatt-med-expressens-karin-olsson/',
         'only_matching': True,
+    }, {
+        'url': 'https://www.expressen.se/tv/livsstil/halsoliv/5-harliga-fakta-om-kaffe/',
+        'md5': '4d4572e17d2bec5fa2c144bb63857934',
+        'info_dict': {
+            'id': '8790448',
+            'ext': 'mp4',
+            'title': '5 härliga fakta om kaffe',
+            'description': 'md5:b7faa986d02765cdd7ccaa0db4673258',
+            'thumbnail': r're:^https?://.*\.jpg$',
+            'duration': 56,
+            'timestamp': 1531986096,
+            'upload_date': '20180719'
+
+        }
+    }, {
+        'url': 'https://www.expressen.se/videoplayer/embed/tv/livsstil/halsoliv/5-harliga-fakta-om-kaffe',
+        'only_matching': True,
+    }, {
+        'url': 'https://www.expressen.se/tv/livsstil/halsoliv/har-gifter-de-sig-pa-10-000-meters-hojd/',
+        'only_matching': True,
     }]
 
     def _real_extract(self, url):
         display_id = self._match_id(url)
 
-        webpage = self._download_webpage(url, display_id)
+        original_url = url
+        embed_url = url.replace('expressen.se/tv/', 'expressen.se/videoplayer/embed/tv/')
+
+        try:
+            webpage = self._download_webpage(embed_url, display_id)
+        except ExtractorError as e:
+            webpage = self._download_webpage(original_url, display_id)
```

Comment:
1. This will try the same URL twice in case of embed URL.
2. Fallback must also apply when no formats was found at first try.

Reply:
@haraldmartin see example in https://github.com/rg3/youtube-dl/blob/master/youtube_dl/extractor/openload.py#L331-L343